Classes/Models/File+Filename.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File+Filename.swift
+//  Freetime
+//
+//  Created by Weyert de Boer on 13/10/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+extension File {
+
+    /// Returns the potential programming language of the file, if unsure it will return `nil`
+    var potentialLanguage: String? {
+        guard let fileExtension = self.filename.lowercased().components(separatedBy: ".").last else { return nil }
+        switch fileExtension {
+        case "swift": return "swift"
+        default:
+            return nil
+        }
+    }
+
+    /// Returns the actual file name
+    public var actualFileName: String {
+        return  self.filename.components(separatedBy: "/").last ?? ""
```

Comment:
Can we remove the double space, and if it fails it should default to just returning `fileName`

Reply:
I have tidied up the code and removed the `potentialLanguage`. I was planning to use that for the 'you know what' PR as Highlight.js is not correctly automatically detected the Swift language 😅

Reply:
Leaving this reply as a test for PR review threads, demonstrating how broken they are 😞